tests/e2e/main_test.go

```diff
@@ -87,7 +87,7 @@ func TestQueryPrometheus(t *testing.T) {
 	}
 
 	// Wait for pod to respond at queries at all. Then start verifying their results.
-	err := wait.Poll(5*time.Second, 1*time.Minute, func() (bool, error) {
+	err := wait.Poll(5*time.Second, 2*time.Minute, func() (bool, error) {
```

Comment:
Why are we increasing this? Do you mind adding the reason to the PR commit as well, thanks!

Reply:
It was a bit short for all the pods to be up which made e2e tests fail half of the time.
I had the same issue while updating release-0.3

Reply:
I detailed the change in the commit message

Reply:
Thanks!